server/serverAPI/model/food/IBaseIngredient.ts

```diff
@@ -2,15 +2,17 @@ export default interface IBaseIngredient {
     /**
      * Unique food identifier.
      */
-    readonly id: number;
+    id: number;
 
     /**
      * Food name.
      */
-    readonly name: string;
+    name: string;
 
     /**
      * Food category.
      */
-    readonly category: string;
+    category: string;
+
+    quantityUnits: string[];
```

Comment:
Add comment please. Also, why is this an array?

Reply:
because there are several units that are available for each ingredient. Spoonacular doesn't allow u to convert anything to anything. In another words, all ingredients have different quantity units available to them. Because of that we store all quantityUnits that are available for specific ingredient in quantityUnits array

Reply:
Can you add that explanation to the comment please:
```
// Array of food quantity units; Spoonacular stores multiple different units for any given ingredient.
```